buildpacks/python/Makefile

```diff
@@ -47,11 +47,11 @@ buildpacks.python.invoker_deps.sha := $(buildpacks.python.temp_dir)/invoker_dep.
 
 $(buildpacks.python.invoker.sha):
 	@mkdir -p $(@D)
-	curl -sw '\0' -L $(buildpacks.python.invoker.url).sha256 -o $@
+	curl -vsw '\0' -L $(buildpacks.python.invoker.url).sha256 -o $@
 
 $(buildpacks.python.invoker_deps.sha):
 	@mkdir -p $(@D)
-	curl -sw '\0' -L $(buildpacks.python.invoker_deps.url).sha256 -o $@
+	curl -vsw '\0' -L $(buildpacks.python.invoker_deps.url).sha256 -o $@
```

Comment:
There's a similar change right above this too

Reply:
It turns out it was pretty useful actually. It helped me debug what error was coming back from the request and ultimately led to the change on line 171.

That said, I don't feel too strongly either way. I thought it could be useful for easier debugging in the future. What's the downside. Does it make our CI logs too verbose?

Reply:
In this case,let's introduce the `--fail` and `--show-error` flags. This will cause curl to not return with status code 0 when it fails to fetch the file.

Reply:
Great suggestion! Done :)